### DISCUSSION TITLE
Downtime notice for search.projectsegfau.lt

### DISCUSSION THREAD

**Discussion by ghost** [searxng/searx-instances#247] [Report instance downtime]
![image](https://user-images.githubusercontent.com/109034945/204036090-c665b1ef-bd37-46f7-b367-eb58c6766cbb.png)
Soleil Levant hosts search.projectsegfau.lt; SearXNG will be affected.

Monitor status at https://status.projectsegfau.lt/status/projectsegfault

**Comment by unixfox** [comment]
Thank you for the notice.
